### PR TITLE
Allow Room Owners to use addhtml

### PR DESCRIPTION
**Pull request by TheMezStrikes**
While writing this I noticed addhtml is not included in the comment decribing all permissions. I'm bad at writing descriptions so I didn't fix it but I thought I'd mention it

### CODE REVIEW

**Comment by xfix** [comment]
Should it `inherit: '*'`, I wonder, what do you think, @Zarel?

File: config/config-example.js [hunk 1]
```diff
@@ -493,6 +493,7 @@ exports.grouplist = [
 		inherit: '@',
```
Comment (xfix) [inline config/config-example.js]: I wonder whether instead of doing this, shouldn't this be changed to inherit from `*` instead of `@`. @Zarel?

**Comment by bumbadadabum** [comment]
Tiny nitpick but you should edit server/rooms.js#L604 because it would now list ROs as normal users. Super niche but needs to be said. Also what xfix said might be a better way to implement this as ROs can promote bots, there's no reason for there to ever be a thing a bot can do that a RO can't.

File: config/config-example.js (hunk 1 above)
Comment (Zarel) [reply config/config-example.js]: I don't want bots part of the regular inheritance chain, even if ROs are supposed to be a strict superset. I've always modeled them as a fork.

**Comment by xfix** [comment]
For a pull request called "Allow Room Owners to use addhtml", it suspiciously doesn't change the room owners permissions, is it intentional?

**Comment by Zarel** [comment]
I mean, I do think this is intentional. Bots can make groupchats and be their host, and being able to control those rooms is probably useful. So it's just the PR title that's wrong.

**Comment by TheMezStrikes** [comment]
Nah that's not intentional, I'm actually stupid and forgot to add the permission to Room Owners

**Comment by Zarel** [comment]
I hope this doesn't backfire...

**Pull request merged**